src/renderer/qssWidget/js/qssMenuWidget.js

```diff
@@ -331,4 +336,15 @@
             container.css(elementStyles);
         }
     };
+    /**
+     * Adds an attribute property for the default setting value.
+     * @param {String} key - The `key` of the setting option.
+     * @param {Object} item - The default value from the settings.
+     * @param {jQuery} container - A jQuery object representing the setting option's container.
+     */
+    gpii.qssWidget.menu.presenter.defaultValue = function (key, item, container) {
+        if (key === item) {
+            container.attr("data-default", true);
```

Comment:
Why a data attribute rather than a CSS class?

Reply:
@amb26 Hey, well, because we noticed that the "selected" state is using attribute (aria-checked I think) too. If its not OK we can change it to class, its not a problem, just let me know.

Reply:
That is a functional attribute referred to the WAI-ARIA standard - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/checkbox_role - DOM manipulations intended to have pure styling effects should use CSS classes, unless we can find a similar standard

Reply:
I can see your point, its changed to class instead.

Reply:
Thanks for the fix - can you also rather than hard-coding the class name in the function block, add it to the "styles" block of the component, alongside the existing "active" style? https://github.com/GPII/gpii-app/blob/664b13ca81c8db61ae8cdc69223bd9c1971a8b8b/src/renderer/qssWidget/js/qssMenuWidget.js#L324
Also while you are doing this could you also namespace the existing bare class name "active" there with the standard prefix .fl-qssMenuWidget used by this component - cheers!

Reply:
Sure, I will do that after today's meeting :-)

Reply:
Done, both of the classes are now in the styles list and have the proper prefix.